test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -10779,6 +10779,141 @@ def forward(self, x):
         x = torch.quantize_per_tensor(torch.randn(3, 4), 0.2, 0, torch.qint8)
         self.run_test(Module(), x)
 
+    @skipIfUnsupportedMinOpsetVersion(13)
+    def test_qat_linear_per_channel(self):
+        class M(torch.nn.Module):
```

Comment:
Is this module the same across all new test cases? If so would it be a good idea to extract the class to the global namespace?

```
class _TestQuatizedModule(torch.nn.Module):
    ...
```